internal/broker/broker.go

```diff
@@ -302,13 +316,13 @@ func (b *Broker) generateUILayout(session *sessionInfo, authModeID string) (map[
 		uiLayout = map[string]string{
 			"type": "qrcode",
 			"label": fmt.Sprintf(
-				"Scan the QR code or access %q and use the code %q",
+				"Scan the QR code or access %q and use the provided code",
 				response.VerificationURI,
-				response.UserCode,
 			),
 			"wait":    "true",
 			"button":  "regenerate QR code",
 			"content": response.VerificationURI,
+			"code":    response.UserCode,
```

Comment:
This good, but if you test this with the current version of gdm where the `code` is not yet supported (I mean the version already in the repository, since the code for supporting it won't land very early in distro and may not be fully synchronized with authd), will it work anyways?

As in the other PR I supported both cases, so both if the UI supports it or not, but I'm not sure if that would fail anywhere. I guess no, since it's optional, but asking for confirmation here.

Reply:
It will work, but the code won't be shown if GDM doesn't support it. The thing is: there's no way to know this on the broker side since what matters here is the supported layouts that authd is sending us (and right now it is sending code as optional). I can attach the code back to the label if you think it's better.